sw/airborne/modules/nav/nav_rotorcraft_hybrid.c

```diff
@@ -118,13 +144,27 @@ static void nav_hybrid_goto(struct EnuCoor_f *wp)
     float max_speed_decel2 = fabsf(2.f * dist_to_wp * nav_max_deceleration_sp); // dist_to_wp can only be positive, but just in case
     float max_speed_decel = sqrtf(max_speed_decel2);
     // Bound the setpoint velocity vector
-    max_h_speed = Min(nav_goto_max_speed, max_speed_decel); // use hover max speed
+    max_h_speed = Min(nav_max_speed, max_speed_decel);
+#ifdef NAV_HYBRID_MIN_DISTANCE_TO_TRANSISTION
+    if (dist_to_wp < NAV_HYBRID_MIN_DISTANCE_TO_TRANSISTION){
+      max_h_speed = Min(nav_goto_max_speed, max_h_speed);
```

Comment:
the #if NAV_HYBRID_FORCE_GOTO_MAX_SPEED is already needed here

Reply:
I think it is worth checking in which case this will actually be useful: if this distance*position_gain is smaller than the nav_goto_max_speed it will never take effect.